devel/libatomicbitvector/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        ekg atomicbitvector e295358fea9532fa4c37197630d037a4a53ddede
+name                libatomicbitvector
+version             2020.05.19
+categories          devel
+license             Apache-2
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Atomic bitset/bitvector with size determined at runtime.
+long_description    This header-only library encodes a bitvector class with size fixed at runtime. \
+                    Atomic operations allow for concurrent access and modification of the bitset. \
+                    Such a structure can help with coordinating parallel processing of a given \
+                    fixed set of entities and has the advantage of only requiring O(1) bit per entry. \
+                    The atomic_bv_t class is a straightforward extension of ConcurrentBitSet \
+                    from Facebook’s folly C++ library. It wraps the atomic type with a class \
+                    that allows them to be copied, and these wrapped atomic types are then used \
+                    to build a vector size of which is determined at runtime.
+homepage            https://github.com/ekg/atomicbitvector
```

Comment:
already set by the `github` PG

Reply:
@reneeotten So just drop the line altogether?